packages/cart/src/types/cart.ts

```diff
@@ -14,12 +14,12 @@ export interface CreateCartDTO {
 
 export interface UpdateCartDTO {
   id: string
-  region_id?: string
-  customer_id?: string
-  sales_channel_id?: string
-  email?: string
-  currency_code?: string
-  metadata?: Record<string, unknown>
+  region_id?: string | null
+  customer_id?: string | null
+  sales_channel_id?: string | null
+  email?: string | null
+  currency_code?: string | null
+  metadata?: Record<string, unknown> | null
```

Comment:
suggestion: To represent the DTO, I wouldn't set them as optional, this should be handled automatically based on the select and relations but not here. If you fetch the cart, you will receive the curren prop marked as optional with a null value but they will be there unless configured differently

Reply:
But this is an UpdateDTO. Isn't that a different case than what  you are describing?

Reply:
My bad, I mixed it with the cart dto, discard